sphinx_gallery/tests/test_full.py

```diff
@@ -861,9 +867,9 @@ def _rerun(
     # - auto_examples/index
     # - auto_examples/plot_numpy_matplotlib
     if how == "modify":
-        n_ch = "[3-7]"
+        n_ch = "[3-9]"
```

Comment:
Could we amend the comment above to add the global execution file to "Ones that can change on stale:". Also I'm lost, why is this not +1 (8) ?

Reply:
Not totally sure why I needed the extra one here but I also think it's okay to be lenient

Reply:
... actually updating the comment made it clear I think, pushing